index.js

```diff
@@ -73,6 +89,28 @@ class Shortcut {
 		 * @type {string}
 		 */
 		this.id = id;
+
+	}
+
+	/**
+	 * Gets the shortcuts metadata from the downloadURL.
+	 * @returns {ShortcutMetadata} An object with all the metadata about the shortcut.
+	 */
+	getMetadata() {
+		return new Promise((resolve, reject) => {
+			got(this.downloadURL, {
+				encoding: null,
+			}).then(response => {
+
```

Comment:
Can you remove the two lines of whitespace between the braces and content of this block?